data/_models/shared/Processes.csv

```diff
@@ -1,119 +1,143 @@
+Disease history,,age of onset,period,,,,,,,,,http://www.ebi.ac.uk/efo/EFO_0004847,"The age at which manifestations of the diseaese first appeared (e.g., ""P32Y6M1D"")","Patient registry, FAIR Genomes"
+Disease history,,age group at diagnosis,ontology,,,CatalogueOntologies,Age groups,,,,,"http://www.ebi.ac.uk/efo/EFO_0004918,http://purl.obolibrary.org/obo/NCIT_C20587","A categorical value that groups this individual's age at diagnosis into groups (e.g., ""Adult"", ""18-24 years"", etc.)","Patient registry, FAIR Genomes"
+Disease history,,age at diagnosis,period,,,,,,,,,http://www.ebi.ac.uk/efo/EFO_0004918,"The age at which the diagnosis was made (e.g., ""P32Y6M1D"")","Patient registry, FAIR Genomes"
+Disease history,,has family history,bool,,,,,,,,,http://purl.obolibrary.org/obo/HP_0032316,An indication if the diagnosis is present in one or more family members or other relatives,Patient registry
+Disease history,,disease severity,ontology,,,CatalogueOntologies,Severity,,,,,http://purl.obolibrary.org/obo/HP_0012824,"A value that describes the intensity of the disease (e.g., ""Profound"")","Patient registry, FAIR Genomes"
+Disease history,,disease stage,ontology,,,CatalogueOntologies,Disease stage,,,,,http://purl.obolibrary.org/obo/OBI_0000278,"A part of an occurrence of a disease process which is associated with position in the normal progression of the disease (e.g., ""acute onset"")","Patient registry, FAIR Genomes"
+Measurements,,,,,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C25209,Additional clinical measurements collected as part of the clinical observation,"Patient registry, FAIR Genomes"
+Measurements,,part of clinical observation,ref,1,TRUE,,Clinical observations,,,,,http://purl.obolibrary.org/obo/NCIT_C87853,"A reference to a specific clinical observation (e.g., ""Ind001_Phen001"")","Patient registry, FAIR Genomes"
+Measurements,,code,ontology,1,TRUE,CatalogueOntologies,Analyses performed,,,,,http://purl.obolibrary.org/obo/NCIT_C25162,"A unique code that identifies or categorises this measurement (e.g., ""Metabolomics"")","Patient registry, FAIR Genomes"
+Measurements,,variable,string,1,TRUE,,,,,,,http://purl.obolibrary.org/obo/NCIT_C54166,"A name that describes the measurement (e.g.. ""Blood platelet count"")","Patient registry, FAIR Genomes"
+Measurements,,value,int,,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C25712,"The value that was measured, collected, derived, etc. (e.g. ""55345"")","Patient registry, FAIR Genomes"
+Measurements,,unit,ontology,,,CatalogueOntologies,Units,,,,,http://purl.obolibrary.org/obo/NCIT_C25709,"A value that specifes the measured value (e.g., ""Per Millimeter"")","Patient registry, FAIR Genomes"
+Metabolomic material processing,Processes,,,,,,,,,,,http://purl.obolibrary.org/obo/OBI_0000073,A metabolomics material processing is a protocol application including material enrollments and biomaterial transformations.,FAIR Genomes metabolomics add-on
```

Comment:
Could become a Metabolomics Assay? I expect the structures for DNA seq, RNA seq, Metabolomics to look similar in concept.

Reply:
Yes, I think so. These tables come from FAIR Genomes, but haven't really be integrated into the RD3 model. I think they aren't loaded by default.